config/exporters/monitoring/grafana/base/kustomization.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+- prometheus-exporter-service-monitor.yaml
 - prometheus-exporter-service.yaml
-images:
-- name: exporter
-  newName: quay.io/redhat-appstudio/o11y
-  newTag: latest
```

Comment:
What's the reason for this change?

Reply:
I removed it thinking newTag was not being used. reverted the changed and renamed the image to exporter:latest